peripherals/syscfg/syscfg_f302.yaml

```diff
@@ -0,0 +1,158 @@
+SYSCFG:
+  CFGR1:
+    I2C3_FMP:
+      Standard: [0, "FM+ mode is controlled by I2C_Pxx_FMP bits only"]
+      FMP: [1, "FM+ mode is enabled on all I2C3 pins selected through selection trhough IOPORT control registers AF selection bits"]
+    ENCODER_MODE:
+      NoRedirection: [0, "No redirection"]
+      # TODO: Naming?
+      MapTim2Tim15: [1, "TIM2 IC1 and TIM2 IC2 are connected to TIM15 IC1 and TIM15 IC2 respectively"]
+      MapTim3Tim15: [2, "TIM3 IC1 and TIM3 IC2 are connected to TIM15 IC1 and TIM15 IC2 respectively"]
+      MapTim4Tim15: [3, "TIM4 IC1 and TIM4 IC2 are connected to TIM15 IC1 and TIM15 IC2 respectively (STM32F302xB/C devices only)"]
+    TIM6_DAC1_DMA_RMP:
+      NoRemap: [0, "TIM6_UP and DAC_CH1 DMA requests mapped on DMA2 channel 3"]
+      Remap: [1, "TIM6_UP and DAC_CH1 DMA requests mapped on DMA1 channel 3"]
+    ADC24_DMA_RMP:
+      NoRemap: [0, "ADC24 DMA requests mapped on DMA2 channels 1 and 2"]
+      Remap: [1, "ADC24 DMA requests mapped on DMA2 channels 3 and 4"]
+    DAC_TRIG_RMP:
+      NoRemap: [0, "DAC trigger is TIM8_TRGO in STM32F303xB/C and STM32F358xC devices"]
+      Remap: [1, "DAC trigger is TIM3_TRGO"]
+    TIM1_ITR3_RMP:
+      NoRemap: [0, "TIM1_ITR3 = TIM4_TRGO in STM32F303xB/C and STM32F358xC devices"]
+      Remap: [1, "TIM1_ITR3 = TIM17_OC"]
+    # TODO: Is documented as reserverd but appears in register table (RM0365 p.172) and register map
```

Comment:
Yea, this looks like an error in the reference manual's longer explanation, I'd leave it in.